stl/inc/format

```diff
@@ -1116,6 +1116,47 @@ _OutputIt _Write(_OutputIt _Out, monostate) {
     return _Out;
 }
 
+// This size is derived from the maximum length of an arithmetic type. The two contenders for widest are double and long
+// long. long long has a max length of ceil(log_10(2^64)) = 20 characters. double has a max length of
+// limits<double>::max_digits10 + the decimal + the sign + e + the exponent's sign + ceil(log_10(DBL_MAX_10_EXP))
+// = 17 + 1 + 1 + 1 + 3 = 24. An example is DBL_MAX which is "1.7976931348623158e+308".
+inline constexpr size_t _Format_min_buffer_length = 24;
+
+// clang-format off
+template <class _CharT, class _OutputIt, class _Arithmetic>
+    requires (is_arithmetic_v<_Arithmetic> && !same_as<_Arithmetic, bool> && !same_as<_Arithmetic, _CharT>)

```

Comment:
I believe this is one of the common constraints in fmt.

Should we create a special concept for it and reuse it ?

Reply:
I don't think (at least in the writing domain) that this will be reused. The other writers which care about specs also care more stringently about type. Maybe we can create a concept for it once it's used again?

Reply:
I think it is, but the current check is in the arg type enum domain and not the real, actual, c++ type domain